### ISSUE TITLE
virError(Code=1, Domain=10, Message='internal error: unable to execute QEMU command 'cont': Resetting the Virtual Machine is required

### ISSUE THREAD

**Issue by frye233** [kubevirt/kubevirt#9816]
**What happened**:
After creating the virtual machine, I entered the noVNC interface to install the system. The installation process may get stuck. After restarting the VM, the installation process will get stuck in another different interface.But when I deploy VM using KVM, there is no situation of getting stuck during the system installation process(Whether multi core or single core).When the number of deployed VM cores is 1, the installation process will not get stuck, the virt-launcher will not report any errors, and the VM can be used normally. When the number of cores exceeds 1, the installation process may become stuck.

virt-launcher log is：

```shell
{"component":"virt-launcher","level":"error","msg":"internal error: unable to execute QEMU command 'cont': Resetting the Virtual Machine is required","pos":"qemuMonitorJSONCheckErrorFull:402","subcomponent":"libvirt","thread":"34","timestamp":"2023-05-29T08:02:29.177000Z"}
{"component":"virt-launcher","kind":"","level":"error","msg":"unpausing the VirtualMachineInstance failed.","name":"centos7","namespace":"default","pos":"manager.go:896","reason":"virError(Code=1, Domain=10, Message='internal error: unable to execute QEMU command 'cont': Resetting the Virtual Machine is required')","timestamp":"2023-05-29T08:02:29.177464Z","uid":"d2773554-27c2-429b-b9bb-36b0c998ff8c"}
{"component":"virt-launcher","kind":"","level":"error","msg":"Failed to sync vmi","name":"centos7","namespace":"default","pos":"server.go:185","reason":"virError(Code=1, Domain=10, Message='internal error: unable to execute QEMU command 'cont': Resetting the Virtual Machine is required')","timestamp":"2023-05-29T08:02:29.177583Z","uid":"d2773554-27c2-429b-b9bb-36b0c998ff8c"}

``` 

kubectl describe vm centos7:
![image](https://github.com/kubevirt/kubevirt/assets/59021773/fba282b1-12bf-4768-b15c-3cdbe8270167)

noVNC:Stuck during system installation
![image](https://github.com/kubevirt/kubevirt/assets/59021773/4191fdd6-1a0e-437e-b9fc-d151307d8c67)

**What you expected to happen**:
The VM starts normally and can install the system normally, and can be used normally after installing the system.

**How to reproduce it (as minimally and precisely as possible)**:
```shell
1. Create a VM with yaml
2. Start the VM
3. VM installation system failed. Viewing pod virt-launcher-centos7-5vbhx logs：virError(Code=1, Domain=10, Message='internal error: unable to execute QEMU command 'cont': Resetting the Virtual Machine is required
``` 

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): v0.58.0
- Kubernetes version (use `kubectl version`): v1.25.0
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): CentOS Linux release 7.6.1810
- Kernel (e.g. `uname -a`): 3.10.0-957.el7.x86_64
- Install tools: N/A
- Others: 
    CPU：Hygon
    qemu-kvm version: 3.1.0
    libvirt version: 5.0.0

centos7.yaml
```shell
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata:
  name: centos7
spec:
  running: true
  template:
    metadata:
      labels:
        kubevirt.io/domain: centos7
    spec:
      domain:
        cpu:
          cores: 4
        devices:
          disks:
          - bootOrder: 2
            cdrom:
              bus: sata
            name: cdromiso
          - bootOrder: 1
            disk:
              bus: virtio
            name: harddrive
          interfaces:
          - bridge: {}
            model: virtio
            name: default
            macAddress: 52:00:00:7c:5b:2c
        machine:
          type: q35
        resources:
          limits:
            memory: 8G
      networks:
      - name: default
        pod: {}
      volumes:
      - name: cdromiso
        persistentVolumeClaim:
          claimName: centos7.6
      - name: harddrive
        hostDisk:
          capacity: 20Gi
          path: /nfs/data/disk_centos7.img
          type: DiskOrCreate

``` 

The system image of the VM to be deployed is CentOS 7.6.1810.

**Comment by alicefr** [comment]
@frye233 it isn't completely clear to me when the system is stuck, is it after the reboot of the VM after the initialization? IIUC, it is working for 1 core but not for multiple?

**Comment by alicefr** [comment]
You also put qemu and libvirt versions. Where are they coming from? The QEMU and Libvirt version should be dependent on the kubevirt version

**Comment by frye233** [comment]
Sorry,my expression may not be very clear.Firstly, it gets stuck during the process of starting the VM for the first time and installing the system, rather than after the reboot of the VM after the installation.Secondly, when I set the number of cores in the VM to 1, the process of starting the virtual machine for the first time and installing the system is normal. And after the system installation is completed, the VM can be used normally. But if I set the number of cores in the virtual machine to multiple, it won't work and it will get stuck.

The QEMU and libvirt versions are physical machines. Because I didn't know that QEMU and Libvirt versions depend on the kubevirt version.I thought upgrading the QEMU and Libvirt versions could solve the problem of getting stuck, but it was not successful.
